CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 ### Bug fixes
 
+### Deprecated functionality
+* Deprecated `skbio.util.flatten`. Please use standard python library functionality
+described here (http://stackoverflow.com/questions/952914/making-a-flat-list-out-of-list-of-lists-in-python)
```

Comment:
:+1: that's a great idea. Probably worth updating the links in the docstring and warning message too. And they'll be shorter! :cake: